### PR TITLE
refactor: incorporate feedback from october session 

### PR DESCRIPTION
**Pull request by meck93**
- remove unnecessary css from implicit grid
- remove named lines section + advanced exercises
- improve speaker notes
- add padding to highlight grid item borders vs. grid border
- increase grid border size

### CODE REVIEW

File: chapters/chapter4.html [hunk 1]
```diff
@@ -649,7 +644,9 @@ <h2>Template Areas (v2)</h2>
           <h2>Further Resources</h2>
           <ul>
             <li>
-              <a href="https://cssgridgarden.com/" target="_blank" rel="noopener">https://cssgridgarden.com/</a>
+              <a href="https://cssgridgarden.com/" target="_blank" rel="noopener"
+                >https://cssgridgarden.com/</a
+              >
```
Comment (culas) [inline chapters/chapter4.html]: We should probably also list the arguably most important resource: https://css-tricks.com/snippets/css/complete-guide-grid/ 😜

Comment (meck93) [reply chapters/chapter4.html]: I'm not sure why it's not already in there. Will add it.

File: chapters/chapter4.html [hunk 2]
```diff
@@ -33,10 +33,10 @@ <h2>Terminology</h2>
...
             <p>
-              The container that holds the entire CSS grid. It is the element that has the
-              display: <span class="code">grid</span>
-              or
-              display: <span class="code">inline-grid</span>
+              The container that holds the entire CSS grid. It is the element that has the display:
+              <span class="code">grid</span>
+              or display:
+              <span class="code">inline-grid</span>
               on it.
```
Comment (mariokaufmann) [inline chapters/chapter4.html]: I stumbled a bit over the phrasing "the display: code on it". Do we maybe want to add "property" here? e.g. "that has the display: grid or display: inline-grid property on it"

**Review by mariokaufmann — APPROVED** [review]
just one small detail, thanks for the update :)

File: chapters/chapter4.html (hunk 2 above)
Comment (meck93) [inline chapters/chapter4.html]:
```suggestion
               property on it.
```

**Comment by meck93** [comment]
**todo**

- remove margin from items. add padd to grid. 


- increase column gap






- maybe add additional slide with visual example

**Pull request merged**